src/pages/index.js

```diff
@@ -6,6 +6,13 @@ import Section from "../components/Section.js";
 import UserInfo from "../components/UserInfo.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import Popup from "../components/Popup.js";
```

Comment:
```js


    this._popupElementCloseBtn.addEventListener("click", () => {
      this.close();
    });
```

Please, delete lines 25-27 from `Popup` because you handle the close icon in line 19

Reply:
completed!